### PR TITLE
Fix grammar mistakes in readme

### PR DESCRIPTION
**Pull request by carnationcrab**
Mostly comma errors with a few others. Should be correct now.

Some grammar notes:

- [This](https://www.businessinsider.com/a-guide-to-proper-comma-use-2013-9) is a really good guide for comma use, but if you want to be a little less technical, my general rule of thumb is to put one in anywhere you would pause while speaking.

- I also changed a few things to active voice-- this isn't super important, but does read a bit better in my opinion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -430,7 +430,7 @@ barion.refundPayment({
 });
 ```
 
-### Send money to bank account - barion.bankTransfer(options, \[callback\])
+### Send money to a bank account - barion.bankTransfer(options, \[callback\])
```
Comment (aron123) [inline README.md]: Can you fix it on the Table of Contents too?

File: README.md [hunk 2]
```diff
@@ -548,18 +548,18 @@ barion.barionTransfer({
 });
 ```
 
-### Handle errors
-There are 3 main types of errors that can be thrown, when you use the ``node-barion`` module:
-  - ``BarionError``: Thrown, when the Barion system responds with errors.
+### Handling errors
```
Comment (aron123) [inline README.md]: Can you fix it on the Table of Contents too?

File: README.md [hunk 3]
```diff
@@ -80,23 +80,23 @@ If you are not familiar with Promise and other ES6 stuff, [get closer to it](htt
 - send money out of Barion via international bank transfer
 - send money to existing Barion account or email address
 
-> **IMPORTANT**: ``node-barion`` is completely consistent with [Barion Docs](https://docs.barion.com/Main_Page), so you can use exactly the same field names, that are specified in it. **It is highly recommended, to read the official Barion documentation** before start to use the ``node-barion`` module.<br>
```
Comment (aron123) [inline README.md]: Is "It is highly recommended" contains grammar mistake? I developed this package myself, and this is not an official module, so "we" is not makes sense for me.

File: README.md (hunk 1 above)
Comment (carnationcrab) [reply README.md]: of course!

File: README.md (hunk 3 above)
Comment (carnationcrab) [reply README.md]: While not strictly grammatically "wrong," it is passive voice which is generally regarded as grammatically inferior. If you would prefer not to use "we" I could switch it back.

**Comment by aron123** [comment]
Thank you.

**Pull request merged**